doto-frontend/src/components/ModalContent.js

```diff
@@ -148,14 +150,19 @@ const ModalContent = props => {
                         </MuiPickersUtilsProvider>
                     </div>
                     <div>
-                        <TextField
-                            className="small-text-area group-spacing"
-                            id="standard-basic"
-                            type="number"
-                            inputProps={{ min: "0" }}
-                            label="Duration (in minutes)"
-                            onChange={handleDurationChange}
-                        />
+                        <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                            <KeyboardTimePicker
+                                ampm={false}
+                                label="Duration of task (hours : minutes)"
+                                margin="normal"
+                                id="time-picker"
+                                value={selectedDuration}
+                                onChange={setselectedDuration}
+                                KeyboardButtonProps={{
+                                    "aria-label": "change time",
+                                }}
```

Comment:
I'm not convinced that using a time picker is the right way to implement selection of duration. Not only does it deviate from the original intent visualised in #198, it also limits the user, making it so they can't input a duration greater than 24 hours. Perhaps that's your intention, that users cannot enter a duration surpassing 24 hours, but it seems to me like a strange limitation. I don't see why something closer to the picture in #198 wasn't implemented.

Reply:
The issue allowed a dial to be used, as well as a limit on the time - I thought 24 hours is more than enough time for a task to be completed in.  The reason as to why I used a dial and not something like in the picture was because there was nothing similar in material-ui and I thought it best to stick to the same theme

Reply:
I suppose it raises another question - what is the purpose of the duration field? Could you give a use case?

Reply:
My cat keeps yelling at me for food and I need to feed her before tomorrow or she'll starve.  I reckon this will take around 15mins.  Due date is tomorrow, but duration is 15mins.

Reply:
Okay, that seems sensible. Thank you for the clarification.